src/content-switcher/content-switcher.component.ts

```diff
@@ -47,7 +47,7 @@ export class ContentSwitcher implements AfterViewInit {
 	/**
 	 * aria-label for the content switcher. Should be set to something descriptive
 	 */
-	@Input() label = "content switcher";
+	@Input() ariaLabel = "content switcher";
```

Comment:
This is, technically, a breaking change.

We should deprecate `label` instead, wrap it with getters and setters to `ariaLabel` and add warning messages in them.